pkg/virtctl/credentials/common/common.go

```diff
@@ -0,0 +1,120 @@
+package common
+
+import (
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/spf13/cobra"
+	v1 "kubevirt.io/api/core/v1"
+)
+
+type CommandFlags struct {
+	User   string
+	Secret string
+}
+
+func (c *CommandFlags) AddToCommand(cmd *cobra.Command) {
+	cmd.Flags().StringVarP(&c.User, userFlag, "u", "", "Name of the user.")
+	err := cmd.MarkFlagRequired(userFlag)
+	if err != nil {
+		panic(err)
+	}
+
+	cmd.Flags().StringVar(&c.Secret, "secret", "", "Name of the secret with SSH keys.")
+}
+
+type SshCommandFlags struct {
+	CommandFlags
+
+	SshPubKeyFile    string
+	SshPubKeyLiteral string
+}
+
+const (
```

Comment:
Place this block inside `AddToCommand`, similar to other `AddToCommand` methods?

Reply:
done

Reply:
Still there?

Reply:
They are used in 2 functions.